ltfnp_executive/src/utils.lisp

```diff
@@ -214,6 +214,26 @@
                                  ("RedMetalCup" (0.1 -0.2) 0.0)))
       (spawn-series-simplified `(("Milk" (-0.05 0.5) 0.0))))))
 
+(defun destination-pose (object-id base-pose)
+  (let ((rel-pose (relative-destination-pose object-id)))
+    (tf:pose->pose-stamped
+     (tf:frame-id base-pose)
+     0.0
+     (cl-transforms:transform-pose
+      (tf:pose->transform base-pose)
+      rel-pose))))
+
+(defun relative-destination-pose (object-id)
```

Comment:
Doesn't defining the relative-destination-pose after its first call in the file (in destination-pose) produce a compiler warning for undefined function? Let's try to avoid that, and move (defun relative-destination-pose ...) in front of (defun destination-pose ...).